src/rgw/rgw_file_int.h

```diff
@@ -2342,21 +2342,18 @@ class RGWStatBucketRequest : public RGWLibRequest,
     return 0;
   }
 
-  virtual int get_params() {
-    return 0;
-  }
+  int get_params(optional_yield) override { return 0; }
 
   void send_response() override {
-    bucket->get_creation_time() = get_state()->bucket->get_info().creation_time;
```

Comment:
I guess I made a blind conclusion based on this. I can't find how `s->bucket` could ever been created as `s->bucket_name` should never been assigned due to a bug(?) here: https://github.com/ceph/ceph/blob/0d56b61200b2d7d1b09e535c767b5f4117678ae3/src/rgw/rgw_os_lib.cc#L34

Reply:
but I'm somehow losing `RGWRequest::s->bucket` after `g_rgwlib->get_fe()->execute_req(&req);`